examples/experimental/dagster-airlift/dagster_airlift/core/airflow_defs_data.py

```diff
@@ -53,3 +57,6 @@ def asset_keys_per_dag(self) -> Mapping[str, AbstractSet[AssetKey]]:
 
     def asset_keys_in_task(self, dag_id: str, task_id: str) -> AbstractSet[AssetKey]:
         return self.asset_keys_per_task_handle[TaskHandle(dag_id=dag_id, task_id=task_id)]
+
+    def get_asset_spec_for_key(self, asset_key: AssetKey) -> AssetSpec:
```

Comment:
seems to not add that much value but nbd.

maybe just name `all_asset_specs_by_key` `asset_specs_by_key` and rely on dictionary access